src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -1092,17 +1149,16 @@ int wm_vuldet_send_cve_report(vu_report *report) {
             if (report->version && *report->version) cJSON_AddStringToObject(j_package, "version", report->version);
             if (report->generated_cpe) cJSON_AddStringToObject(j_package, "generated_cpe", report->generated_cpe);
             if (report->arch && *report->arch) cJSON_AddStringToObject(j_package, "architecture", report->arch);
-            if (!report->pending) {
-                if (report->condition && *report->condition != '\0') {
-                    cJSON_AddStringToObject(j_package, "condition", report->condition);
-                } else if (report->operation_value) {
-                    free(report->condition);
-                    os_calloc(OS_SIZE_1024 + 1, sizeof(char), report->condition);
-                    snprintf(report->condition, OS_SIZE_1024, "Package %s %s", report->operation, report->operation_value);
-                    cJSON_AddStringToObject(j_package, "condition", report->condition);
-                } else {
-                     cJSON_AddStringToObject(j_package, "condition", report->operation);
-                }
+
```

Comment:
This caused the condition to be added to the alert only when a fix was already released. Now, adding the `condition` in any acse.